objcgen/TypeMapper.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Embeddinator
+{
+	public class TypeMapper
+	{
+		Dictionary<ProcessedType, Dictionary<string, ProcessedMemberBase>> MappedTypes = new Dictionary<ProcessedType, Dictionary<string, ProcessedMemberBase>> ();
+		
+		IEnumerable<string> GetNames (ProcessedMemberBase member)
+		{
+			if (member is ProcessedMemberWithParameters) {
+				yield return ((ProcessedMemberWithParameters)member).ObjRawCSignature;
```

Comment:
https://www.danielcrabtree.com/blog/152/c-sharp-7-is-operator-patterns-you-wont-need-as-as-often

It's even easier with c#7 to avoid those :)